docs/pages/guides/expo-sdk-explained.md

```diff
@@ -0,0 +1,74 @@
+
+## Significance of the Expo SDK in:
+
+### Expo Go
+
+Expo Go is where your Expo SDK version is most important. Whether you are in the bare workflow or the managed workflow, if you use Expo Go for development, you should be aware of your Expo SDK version and use `expo install` when you install dependencies.
+
+We version the native code in Expo Go so that you can run projects on different SDK versions easily, but this means you need to tell Expo Go which version of native code to run. This is usually done via the `expo` package in your `package.json` (unless you override it with the `sdkVersion` property in `app.json`), which points to a particular SDK version (e.g. "^41.0.0").
+
+### Non-Expo development
+
+If you are developing with `react-native cli` and without Expo Go, then the Expo SDK version has very little impact on your development workflow. We recommend keeping up with our SDK releases just so you're aware of new features and changes, but you can install packages normally without `expo install`, and the native code for your project is defined by the packages you have installed themselves, rather than by the Expo SDK version. This also means you can edit the native code files and see those changes in your project.
+
+### EAS Build
+
+[EAS Build](/build/introduction.md) is similar to Non-Expo development- the native code for your project is defined by your individual dependencies, rather than by the Expo SDK version. Additionally, unlike [Classic `expo build`](), _only_ the native code for the dependencies you have excplicitly installed is included with your build. This has a lot of good consequences, like:
+
+- Smaller app size 📦
+- Ability to leave out unwanted code 🙅
+- Ability to add in libraries that are not included in the Expo SDK 🤝
+- Ability to install private packages 🤫
+
+but also comes with some caveats:
+
+- The entire SDK is **no longer included by default**, so you'll need to ship a new binary when you add in a new Expo package. For example, if you'd like to add camera functionality to your app, but didn't have `expo-camera` as a dependency during your last build, you cannot add in `expo-camera` via an OTA Update like you could previously when using Classic `expo build`. Instead, you'll need to install `expo-camera`, and rebuild with `eas build`.
+- Similar to the point above, it is easier to ship an incompatible OTA Update to your app, so you'll need to be aware of that.
+
+### Classic expo build
```

Comment:
I think the heading here and for EAS Build above should be similar, either both should be the CLI command or both should be brand names.

Reply:
agreed, I just wasn't sure of the brand name. I think it's "Classic Build"?

Reply:
yeah I think it's not really known as this widely at the moment but we could start using that phrasing. i feel like it's worth qualifying if we say "Classic Build" to "Classic Build (`expo build`)" - this isn't so necessary for eas build because the name is the same as the command